less/navigation.less

```diff
@@ -1 +1,41 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+.nav-bar {
+    max-width: 1000px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    margin: 10px auto 0 auto;
+    padding-bottom: 20px;
+    justify-content: space-around;
```

Comment:
I would use space-between here which will push the logo and the nav to either edge of your header